docs/content/en/docs/clustermgmt/etcd-backup-restore/ubuntu-rhel-etcd-backup.md

```diff
@@ -40,7 +41,10 @@ chown ec2-user snapshot.db
 scp -i $PRIV_KEY ec2-user@$ETCD_VM_IP:/home/ec2-user/snapshot.db . 
 ```
 
-NOTE: This snapshot file contains all information stored in the cluster, so make sure you save it securely (encrypt it).
+NOTE: 
+1. This snapshot file contains all information stored in the cluster, so make sure you save it securely (encrypt it).
+2. Following steps are for EKS-A on vSphere provider, where the node username is ec2-user. If you are running these backup and restore commands on a different provider,
```

Comment:
"The following steps are for EKS Anywhere on vSphere where the node username is `ec2-user`."